src/ScrumBoardItBundle/Security/AbstractTokenAuthenticator.php

```diff
@@ -29,10 +29,10 @@ public function __construct(Router $router, $data, $rememberme)
 
     public function getCredentials(Request $request)
     {
-        if ($request->getPathInfo() != '/login_check' || ! $request->isMethod('POST')) {
+        // Check if request comes from the login form and if the requested api is the current one
+        if ($request->getPathInfo() != '/login_check' || ! $request->isMethod('POST') || $this->getApi() != $request->request->get('_api')) {
```

Comment:
if $request->getPathInfo() equal 'toto', the test is true; Are you sure about this test?